src/simtools/simtel/simulator_ray_tracing.py

```diff
@@ -137,7 +141,13 @@ def _load_required_files(self, force_simulate):
             self._write_out_single_pixel_camera_file()
 
     def _make_run_command(self, run_number=None, input_file=None):  # pylint: disable=unused-argument
-        """Generate sim_telarray run command."""
+        """
+        Generate simtel_array run command. Export sim_telarray configuration file(s).
+
+        Parameters are not relevant for the ray tracing simulation.
```

Comment:
I suggest to be a little more specific. That is "The `run_number` and `input_file` parameters are not relevant for the ray tracing simulations and are given as arguments only to comply with the structure of the parent class function" (if I even understood it correctly...)